src/System.Management.Automation/engine/hostifaces/MshHostUserInterface.cs

```diff
@@ -234,7 +234,7 @@ public virtual void WriteLine(ConsoleColor foregroundColor, ConsoleColor backgro
 
         private static bool ShouldOutputPlainText(bool isHost, bool? supportsVirtualTerminal)
         {
-            var outputRendering = OutputRendering.Ansi;
+            var outputRendering = OutputRendering.PlainText;
 
             if (supportsVirtualTerminal != false)
```

Comment:
Nit:
```suggestion
            if (supportsVirtualTerminal)
```

Reply:
It's of the `bool?` type, so `!= false` is required.

Reply:
It is `public virtual bool SupportsVirtualTerminal { get { return false; } }`

Reply:
The parameter is `bool? supportsVirtualTerminal`, you can check out how this method is used.

Reply:
> you can check out how this method is used.

I just do not see why it is made nullable - bool is enough in all methods.

Reply:
Well, when it's not used by a host (`isHost == false`), I guess `supportsVirtualTerminal` doesn't make sense?
But feel free to open an issue for discussion if you want.

Reply:
If follow the logic we should remove isHost parameter at all, shouldn't we? :-)